Mathlib/LinearAlgebra/TensorProduct/Basis.lean

```diff
@@ -66,5 +67,181 @@ lemma Basis.baseChange_apply (b : Basis ι R M) (i) :
     b.baseChange S i = 1 ⊗ₜ b i := by
   simp [Basis.baseChange, Basis.tensorProduct]
 
+variable (K : Type*) [CommSemiring K] (B C: Type*) [AddCommMonoid B] [Module K B]
+    [AddCommMonoid C] [Module K C]
```

Comment:
All the previous results were in a section and used completely different variables. The point of sections is to control the scope of variables. Probably you should have let the section end here and started a new one if you had no interest in the variables already set up. Note also that all your lemmas have variables `{ιC ιB: Type*} (𝒞 : Basis ιC K C) (ℬ : Basis ιB K B)` so perhaps they should also be variables in your new section.

Reply:
good point I've adapted the original variable names now